testplan/report/testing/schemas.py

```diff
@@ -67,14 +77,132 @@ def _deserialize(self, value, attr, data):
         })
 
 
+class EntriesField(fields.Field):
+    """
+    Handle encoding problems gracefully
+    """
+
+    _BYTES_KEY = 'bytes'
```

Comment:
I think that this key should maybe be mangled a bit e.g. "_bytes" - otherwise, what if an object we are comparing already has a field named "bytes"? Of course, whatever we set it to there is still some risk of collision, but we can at reduce that risk.